### DISCUSSION TITLE
Add Vue DevTools

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#3663] [Ideas]
We currently integrate with React DevTools out of the box. It would be great to offer a similar integration for Vue.

https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd?hl=en
https://devtools.vuejs.org/
![vue](https://user-images.githubusercontent.com/254562/133914736-9246f344-7841-491e-8963-801950747737.jpeg)

**Comment by jasonLaster** [comment]
cc @JessicaSachs

**Reply by JessicaSachs** [reply]
Heh. I think there's been some progress by Vue Core team and this is doable with the `next` version of vue-devtools. This is something we wanted to do with Cy but ended up deprioritizing. I don't have enough time to pick it up again.

**Reply by jasonLaster** [reply]
Got it. Yeah - we basically piggy backed off of codesandbox support. I could see cypress being fairly similar
